package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should be able to remove scripts and dependencies.

Reply:
done